src/test/java/com/spotify/docker/client/DefaultDockerClientTest.java

```diff
@@ -263,9 +264,12 @@ public void testMissingAuthParam() throws Exception {
   @Test
   public void testInfo() throws Exception {
     final Info info = sut.info();
+    assertThat(info.driverStatus(), any(List.class));
```

Comment:
Should we have these checks? Older docker API versions might not return these fields.